### ISSUE TITLE
Broken link on zetterbergjansenritmodel

### ISSUE THREAD

**Issue by rgerkin** [OpenSourceBrain/OSB_Status#7]
GitHub link on http://www.opensourcebrain.org/projects/zetterbergjansenritmodel is broken.

**Comment by pgleeson** [comment]
@rgerkin That's true... There were a few projects set up to look at interactions between The Virtual Brain and OSB, including this one, which never really got off the ground, due to busy people on both sides. Hopefully a Google Summer of Code student at some point can help with these...

Marking the project as a 'User Project' as opposed to OSB endorsed to signal it's not supported. @maedoc, @pausz let me know if you'd like to revive this at some point.

**Comment by pausz** [comment]
@pgleeson Thank you for letting us know and sorry about the leftover 'dead' projects. I still hope someone will get around incorporating mesoscopic models into OSB.
